tests/shell_sec/main.c

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright (C) 2017 OTA keys S.A.
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @file
+ * @brief       shows how to set up and use shell_sec
+ *
+ * @author      Vincent Dupont <vincent@otakeys.com>
+ *
+ */
+
+#include <stdio.h>
+#include <string.h>
+
+#include "shell_commands.h"
+#include "shell_sec.h"
+#include "shell.h"
+
+#define NB_USER     2
+#define USER_LENGTH 10
+
+static char _user[10];
+
+static char user_list[NB_USER][USER_LENGTH] = {
+    "user",
+    "admin",
+};
+
+static char passwd_list[NB_USER][USER_LENGTH] = {
+    "passwd1",
```

Comment:
what about using a list of encrypted passwords and compare the encrypted user entry with them ?